draft-quic-atsss-reqs.mkd

```diff
@@ -122,8 +122,8 @@ This steering can be managed by the smartphone, but flows need to continue
 after a handover. 
 
 Other smartphones users may want to combine the cellular and the Wi-Fi network
-to improve the performance of individual flows. The GigaLTE service deployed
-using Multipath TCP in South Korea in an example of this use case {{IETFJ16}}.
+to improve the up and download throughput performance of individual flows. 
+The GiGA LTE and GiGA 5G service deployed using Multipath TCP in South Korea in an example of this use case {{IETFJ16}}.
```

Comment:
I am fine with those details

Reply:
I close the pull request and adopt the update to the gigalte description